src/lib/queries.js

```diff
@@ -0,0 +1,10 @@
+import { Q } from 'cozy-client'
+
+export const buildMyselfQuery = () => {
+  return {
+    definition: Q('io.cozy.contacts').where({ me: true }),
+    options: {
+      as: 'io.cozy.contacts/myself'
```

Comment:
does it work with `singleDocData: true`? You will not need `const me = data[0]` then

Reply:
It works only with and `_id` for the moment https://github.com/cozy/cozy-client/issues/1350